source/game/field/RKGndCol.cc

```diff
@@ -0,0 +1,374 @@
+#include "RKGndCol.hh"
+
+#include <cmath>
+
+#include "egg/math/Math.hh"
+
+#include "game/field/KCollisionTypes.hh"
+
+// Some functions from m8 -
+// https://github.com/em-eight/mkw/tree/master/source/game/field/RKGndCol.cpp
```

Comment:
If we want to include mentions like this, should we also do this for stebler's code?

Reply:
Yes, we absolutely should. Although I think we should credit as `em-eight/mkw` or `stblr/Hanachan` to avoid fixed filepaths, as they may deviate.